website_sale_coupon_page/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Website Sale Coupon Page",
+    "version": "13.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/sale-promotion",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["website_sale_coupon"],
```

Comment:
I think you only need to depend on `website` and `sale_coupon`, not on this one.